frontend/test/unit/specs/components/VHeader/v-header-desktop.spec.js

```diff
@@ -50,4 +50,37 @@ describe("VHeaderDesktop", () => {
       searchType: "all",
     })
   })
+
+  it("sends TOGGLE_FILTER_SIDEBAR analytics event when the filter sidebar is opened", async () => {
+    const screen = render(VHeaderDesktop, options)
+    const filterSidebarTrigger = screen.getByText("Filters")
+
+    await fireEvent.click(filterSidebarTrigger)
+
+    expect(sendCustomEventMock).toHaveBeenCalledWith("TOGGLE_FILTER_SIDEBAR", {
+      searchType: "all",
+      toState: "opened",
+    })
+  })
+
+  describe("when the filter sidebar is visible", () => {
+    beforeEach(() => {
+      options.provide[IsSidebarVisibleKey] = ref(true)
```

Comment:
Nice solution for not having to click "Filters" button twice!

Reply:
Thanks!